### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Timi2424**
[DEMO LINK](https://Timi2424.github.io/react_list-of-goods/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -26,70 +26,122 @@ type ReorderOptions = {
...
+    visibleGoods.sort((a, b) => a.length - b.length);
+  }
+
+  if (isReversed) {
+    visibleGoods = [...visibleGoods].reverse();
+  }
+
+  if (sortType === SortType.NONE) {
+    visibleGoods = [...visibleGoods];
+  }
```
Comment (zaura333) [inline src/App.tsx:49]: This condition isn't necessary here, because visibleGoods are already a copy of original array

**Review by zaura333 — APPROVED** [review]
Looks good! just one suggestion

File: src/App.tsx (hunk 1 above)
Comment (Timi2424) [reply src/App.tsx:49]: you are right, thank you!

File: src/App.tsx [hunk 2]
```diff
@@ -26,70 +26,118 @@ type ReorderOptions = {
...
+    return (
+      <div className="section content">
+        <div className="buttons">
+          <button
+            type="button"
+            className={
+              this.state.sortType === SortType.ALPHABET
+                ? 'button is-info'
+                : 'button is-info is-light'
+            }
```
[the remark this answers is not in the document]
Comment (Timi2424) [reply src/App.tsx:94]: Looking good!

**Review by mvjl000 — APPROVED** [review]

**Pull request closed without merging**